keyboards/handwired/jtallbean/split_65/info.json

```
@@ -0,0 +1,88 @@
+{
+    "keyboard_name": "split_65",
+    "url": "",
+    "maintainer": "jtallbean",
+    "width": 18.25,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
```

As the board supports multiple layouts, this should be named `LAYOUT_all`.

```suggestion
        "LAYOUT_all": {
```